apps/marketplace/src/global.css

```diff
@@ -8,11 +8,14 @@ html {
   scroll-behavior: smooth;
 }
 body {
+  color: #ffffff;
+  display: flex;
+  flex-direction: column;
   font-family: inherit;
+  justify-content: space-between;
```

Comment:
Added these to line up the footer, but will probably need to be updated once page content is displayed.